<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MalzemeGirDialog</class>
 <widget class="QDialog" name="MalzemeGirDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>334</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QLabel" name="label_18">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Malzeme Girişi&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Malzeme Bilgiler</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="7" column="1">
           <widget class="QDoubleSpinBox" name="doubleSpinBox_birimFiyati">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="maximum">
             <double>99999999.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLineEdit" name="lineEdit_MalzemeAdi"/>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_SelectMalzeme">
              <property name="text">
               <string>+</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLineEdit" name="lineEdit_Kategori"/>
            </item>
           </layout>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Kategori</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Malzeme Adı</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>Miktar</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QDoubleSpinBox" name="doubleSpinBox_miktar">
            <property name="maximum">
             <double>99999999.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QComboBox" name="comboBox_selectDepo"/>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>Toplam Tutar</string>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>Birim Fiyatı</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QLineEdit" name="lineEdit_birimi">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>Birimi</string>
            </property>
           </widget>
          </item>
          <item row="8" column="1">
           <widget class="QDoubleSpinBox" name="doubleSpinBox_ToplamTutar">
            <property name="maximum">
             <double>100000000000000000.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Malzeme Kodu</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEdit_malzemeKodu">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Depo</string>
            </property>
           </widget>
          </item>
          <item row="9" column="0">
           <widget class="QLabel" name="label_16">
            <property name="text">
             <string>İskonto  indirimi</string>
            </property>
           </widget>
          </item>
          <item row="9" column="1">
           <widget class="QDoubleSpinBox" name="doubleSpinBox_indirimMiktari">
            <property name="maximum">
             <double>99999999999999.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="10" column="1">
           <widget class="QDoubleSpinBox" name="doubleSpinBox_KDV">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="font">
             <font>
              <kerning>true</kerning>
             </font>
            </property>
            <property name="maximum">
             <double>9999999999999.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="10" column="0">
           <widget class="QLabel" name="label_17">
            <property name="text">
             <string>KDV</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_19">
            <property name="text">
             <string>Kategori Kodu</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEdit_kategoriKodu">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Fatura Bilgileri</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="3" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>Seri</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_faturaSeri"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>No</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_faturaNO"/>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item row="6" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLineEdit" name="lineEdit_CariKodu">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton">
              <property name="text">
               <string>+</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Fatura No</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Fatura Tarihi</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QComboBox" name="comboBox_faturaTipi">
            <item>
             <property name="text">
              <string>iade</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Normal</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QDateEdit" name="dateEdit_faturaTarihi">
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Fatura Tipi</string>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Firmanın Adı</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Firmanın Cari Kodu</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QLineEdit" name="lineEdit_FirmaAdi">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="8" column="1">
           <widget class="QLineEdit" name="lineEdit_FirmaAdi_oid">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="label_20">
            <property name="text">
             <string>Firmanın Oid</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPushButton" name="pushButton_cancel">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 0, 0);
font: 75 10pt &quot;Tahoma&quot;;
text-decoration: underline;</string>
       </property>
       <property name="text">
        <string>İptal</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_save">
       <property name="styleSheet">
        <string notr="true">font: 75 10pt &quot;Tahoma&quot;;
text-decoration: underline;
color: rgb(85, 200, 127);</string>
       </property>
       <property name="text">
        <string>Kaydet</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_status">
     <property name="text">
      <string>status</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
